plugins/services/src/js/structs/Pod.js

```diff
@@ -149,6 +149,18 @@ module.exports = class Pod extends Service {
     return this._spec;
   }
 
+  /**
+   * Returns the amount of running instances, wheather they are healthy or not
+   * @override
+   */
+  getTaskCount() {
```

Comment:
There is already a method in this struct, `countRunningInstances`, that does the same thing as this. So I don't think this is needed?